### PR TITLE
fix: 🐛 display overview incase of unbonding and redeemable bal

### PR DESCRIPTION
**Pull request by sahilnanda1995**

### CODE REVIEW

File: components/overview/index.js [hunk 1]
```diff
@@ -278,7 +278,11 @@ const Overview = () => {
 				</span>
 			</div>
 		</div>
-	) : isNil(allNominationsData) && isNil(userData) ? (
+	) : isNil(allNominationsData) &&
+	  isNil(userData) &&
+	  unbondingBalances.length == 0 &&
+	  isNil(unlockingBalances) &&
+	  redeemableBalance == 0 ? (
```
Comment (saumyakaran) [inline components/overview/index.js:285]: Request to change `redeemableBalance == 0` to `redeemableBalance === 0`

**Review by saumyakaran — CHANGES_REQUESTED** [review]

**Review by saumyakaran — APPROVED** [review]
Approving based on @sahilnanda1995's judgement + assuming that the [requested changes](https://github.com/yieldscan/yieldscan-frontend/pull/7/files/4f0cc87d162f5e7ff5b9d986cd4084aaae9d210d) will be fixed before merging.

File: components/overview/index.js [hunk 2]
```diff
@@ -278,7 +278,11 @@ const Overview = () => {
 				</span>
 			</div>
 		</div>
-	) : isNil(allNominationsData) && isNil(userData) ? (
+	) : isNil(allNominationsData) &&
+	  isNil(userData) &&
+	  unbondingBalances.length == 0 &&
```
[the remark this answers is not in the document]
Comment (sahilnanda1995) [reply components/overview/index.js:283]: :)

File: components/overview/index.js (hunk 1 above)
Comment (sahilnanda1995) [reply components/overview/index.js:285]: :)

**Pull request merged**